### ISSUE TITLE
Run subtensors

### ISSUE THREAD

**Issue by ppolewicz** [backend-developers-ltd/auto-validator#6]

**Comment by sirouk** [comment]
Any plans to collaborate with SubVortex on this? Even for optimal subtensor deployment suggestions/scripting?

**Comment by ppolewicz** [comment]
not really, validators will roll out their own. Subtensor quality is too important and subvortex too unstable.
